udagram-api-feed/src/aws.ts

```diff
@@ -3,8 +3,10 @@ import {config} from './config/config';
 
 
 // Configure AWS
-const credentials = new AWS.SharedIniFileCredentials({profile: config.aws_profile});
-AWS.config.credentials = credentials;
+AWS.config.update({
```

Comment:
For the sake of coherece, put this into config, and the use the config value as we do in other places. The change is fine :)